lib/oxidized/input/ftp.rb

```diff
@@ -1,25 +1,21 @@
 module Oxidized
   require 'net/ftp'
   require 'timeout'
-  require_relative 'cli'
-
+  require 'oxidized/input/cli'
   class FTP < Input
     RescueFail = {
-      :debug => [
```

Comment:
I know it's mainly just comments that's been removed but any reason you've done that? It may be helpful to others when reading the code.